local_devnet/docker-compose.yml

```diff
@@ -39,7 +39,7 @@ services:
       - core0
     environment:
       - MONIKER=core1
-      - CELESTIA_HOME=/opt
+      - CELESTIA_APP_HOME=/opt
```

Comment:
It's unclear to me why `CELESTIA_HOME` was `/opt` in these files. That sounds like the install directory, not the directory to place data.

Reply:
if I remember correctly, at the day of creating these scripts, a couple years ago, I had some issues with alpine and the home directory and used `/opt`. Can definitly be changed

Reply:
Since you're familiar with this script, can you make an issue to follow-up on this? I'll leave it as it is in the PR since it's orthogonal.

Reply:
https://github.com/celestiaorg/celestia-app/issues/4670